marquez_airflow/extractors/bigquery_extractor.py

```diff
@@ -72,16 +62,45 @@ def _bq_table_name(self, bq_table):
     # convenience method
     def extract_on_complete(self, task_instance) -> [StepMetadata]:
         log.debug(f"extract_on_complete({task_instance})")
-
-        source = self._source()
-        bigquery_job_id = self._get_bigquery_job_id(task_instance)
-
         inputs = None
         outputs = None
         context = {
             'sql': self.operator.sql,
-            'bigquery.job_id': bigquery_job_id
```

Comment:
Why are we removing this information? This will be useful in the future if we want to link to the bigquery dashboard for example.